lib/graph_records.rb

```diff
@@ -74,7 +287,11 @@ def render_edges
     @edges.map { |from, to| "  #{node_name(from)} --> #{node_name(to)}" }
   end
 
-  def inspect(obj)
+  def render_clicks
+    @nodes.to_a.map { "  click #{node_name(it)} \"#{node_link(it)}\"" }
```

Comment:
Also don't need the `to_a` here.

```suggestion
    @nodes.map { "  click #{node_name(it)} \"#{node_link(it)}\"" }
```